packages/build/tests/plugins/load/snapshots/tests.js.md

```diff
@@ -36,27 +36,25 @@ Generated by [AVA](https://ava.li).
     ␊
     ␊
     ┌─────────────────────────────┐␊
-    │     Netlify Build Error     │␊
+    │        Build failed         │␊
     └─────────────────────────────┘␊
     ␊
-    Plugin "netlify-plugin-test" failing with errors␊
+    > Error message␊
```

Comment:
Need to specify the error is from a plugin. This was a big ask from support & DX team